Rakefile

```diff
@@ -11,4 +11,15 @@ end
 
 RuboCop::RakeTask.new
 
+task :test do
+  ENV["MD_LOAD_MODE"] = "inline"
+  $stdout.puts "⚙️ Runs rubocop with '-r rubocop_md' options"
+  Rake::Task[:rubocop_md_tests].invoke
+
+  ENV["MD_LOAD_MODE"] = "config"
+  $stdout.puts "⚙️ Runs rubocop with 'required rubocop_md' section in .rubocop.yml"
+  Rake::Task[:rubocop_md_tests].reenable
+  Rake::Task[:rubocop_md_tests].invoke
```

Comment:
We should define individual tasks and then an aggregate using Rake dependencies.

The main reason is that we need a way to run each mode individually.